tomviz/python/AutoCrossCorrelationTiltImageAlignment.py

```diff
@@ -66,6 +66,12 @@ def transform_scalars(self, dataset):
 
         utils.set_array(dataset, tiltSeries)
 
+        # Assign Negative Shifts when Shift > N/2.
+        indeces_X = np.where(offsets[:, 0] > tiltSeries.shape[0] / 2)
```

Comment:
I think you mean indexes or indices as the prefix for the variable names, updating would make this more readable.